star-navigation/src/utils/sim-utils.ts

```diff
@@ -82,6 +82,15 @@ export const toPositiveHeading = (heading: number) => {
   return (360 - heading) % 360;
 };
 
+export const toNegativeHeading = (heading: number) => {
+  // Opposite of toPositiveHeading
+  if (heading > 180) {
+    // [180, 360] range representing N-E-S side
+    return 360 - heading;
```

Comment:
Modulus needed here?

Reply:
I think not this time, as modulus is pretty much used to show 0 instead of 360 in some other cases (while both are correct in fact, but just for consistency with the view). Here we have `360 - heading`, but `heading > 180`, so there's no way to end up with `360`.